config/kubernetes/kustomization.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: coco-guest
+
+images:
+- name: kbs-container-image
+  newName: quay.io/surajd/kbs-builtin-as
+  newTag: latest
+
+resources:
+- namespace.yaml
+- deployment.yaml
+- service.yaml
+
+configMapGenerator:
+# KBS configuration.
+- name: kbs-config
+  files:
+  - kbs-config.json
+
+secretGenerator:
+# KBS auth public key.
+- name: kbs-auth-public-key
+  files:
+  - kbs.pem
+# Store keys that KBS will release to workloads after attestation.
+# /reponame/workload_key/key.bin
+- name: keys
+  files:
+  - key.bin
```

Comment:
Why do we need this key?

Reply:
This is just an example secret that user can provide. They can call it whatever they want.

Reply:
So this is a secret that could be retrieved by cc-kbc right? We might need a note to show the way to access this secret, s.t. a URI like `kbs:///reponame/workload_key/key.bin`